games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
 categories              games
 platforms               darwin
-maintainers             nomaintainer
+maintainers             @Zweihorn
```

Comment:
at least add "openmaintainer" as well so that others can make minor changes to the Portfile. 

Again, an e-mail address is highly recommended and you can just make a dedicated e-mail if you want. 

I am pretty sure you'll not get commit access and a macports.org address as that requires a track record of contributions as described in the guide.

Reply:
No problem. Again I just followed the explanations from the MacPorts Guide, I presume.

The addition of an `openmaintainer` clearly would help.

Furthermore, my commitment is just to help in a community effort, and _certainly **not** to rule them all ..._

Reply:
Regarding the **email** we could wait and see what happens to my application at MacPorts Mgt